docs/proposals/image-prepull_expand.md

```diff
@@ -0,0 +1,252 @@
+# Image PrePulling Feature Enhancement
+
+## Motivation
+
+### Goals
+
+In the previous version, each task execution currently only supports images from the same image warehouse, and only supports the configuration of one secret. Now, we will enhance this function, which will support covering images and secrets, and can configure multiple secrets to pull images from multiple warehouses.
+
+
+
+## Design details
+
+### Architecture
+
+![kubeedge镜像预拉取增强](../images/proposals/image-prepull-expand.png)
+
+
+
+### Ideas
+
+After generating Secrets for different warehouses in advance through Kubectl secret, use the Secret and Image correspondence directly in the API. The Job will perform an apply operation on the edge, pull the image to generate a Pod; after the previous step is completed, delete the generated Pod, that is This ensures that images from different image warehouses are loaded into corresponding nodes.
+
+
+
+### Image PrePullController
+
+`ImagePrePullController` is responsible for dealing with ImagePrePullJob, watches the resource with list-watch and sends image prepull message to edgenodes.
+
+- Use List-Watch mechanism to monitor ImagePrePullJob CRD resources, after receiving events from K8s APIServer, then store it in local cache using map.
+- Use K8s informer to get node list according to NodeNames or LabelSelector specified in CR, and filter out nodes that don't meet image prepull requirements(1. not edge node, without label "node-role.kubernetes.io/edge": "" 2. edge node is in NotReady state 3. remove duplicated nodes)
+- Deal with the status reported from edge nodes and update the ImagePrePullJob.
+
+
+
+### override manager
```

Comment:
What is override manager? Is it a module? What the capability of override manager and which fields can be overridden by the overmanager?